docroot/modules/custom/mass_fields/mass_fields.deploy.php

```diff
@@ -53,3 +53,91 @@ function mass_fields_deploy_search_fields() {
+  $show_parent_nids = [705346, 408791, 790226, 342871, 514371, 415871, 685541, 468571, 22826, 51951, 418061, 64866, 571991, 646046, 790506, 14686, 683761, 13626, 47751, 528126, 15091, 74746, 629876, 609336, 71546, 144916, 71416, 74466, 6801, 511121, 13726, 409611, 507841, 74116, 13891, 205351, 27511, 488491];
+  $nids_to_load = array_unique(array_merge($do_not_include_search_nids, $show_parent_nids));
+
+  $query = \Drupal::entityQuery('node')->accessCheck(FALSE);
+  $query->condition('type', 'org_page');
+  $query->condition('nid', $nids_to_load, 'IN');
+
+  if (empty($sandbox)) {
+    // Get a list of all nodes of type org_page.
+    $sandbox['progress'] = 0;
+    $sandbox['current'] = 0;
+    $count = clone $query;
+    $sandbox['max'] = $count->count()->execute();
+  }
+
+  $batch_size = 50;
+
+  $nids = $query->condition('nid', $sandbox['current'], '>')
+    ->sort('nid')
+    ->range(0, $batch_size)
+    ->execute();
+
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
+
+  $nodes = $node_storage->loadMultiple($nids);
+
+  // Turn off entity_hierarchy writes while processing the item.
+  \Drupal::state()->set('entity_hierarchy_disable_writes', TRUE);
+
+  foreach ($nodes as $node) {
+    $sandbox['current'] = $node->id();
+
+    try {
+      if (in_array($node->id(), $do_not_include_search_nids)) {
+        $node->set('field_org_no_search_filter', 1);
+      }
+      if (in_array($node->id(), $show_parent_nids)) {
+        $node->set('field_include_parent_org_search', 1);
+      }
+      $node->setSyncing(TRUE);
+      $node->save();
+    }
+    catch (\Exception $e) {
+      \Drupal::state()->set('entity_hierarchy_disable_writes', FALSE);
+    }
+    if (!$node->isLatestRevision()) {
+      $storage = \Drupal::entityTypeManager()->getStorage('node');
+      $query = $storage->getQuery()->accessCheck(FALSE);
+      $query->condition('nid', $node->id());
+      $query->latestRevision();
+      $rids = $query->execute();
+      foreach ($rids as $rid) {
+        $latest_revision = $storage->loadRevision($rid);
+        if (isset($latest_revision)) {
+          try {
+            if (in_array($latest_revision->id(), $do_not_include_search_nids)) {
+              $latest_revision->set('field_org_no_search_filter', 1);
+            }
+            if (in_array($latest_revision->id(), $show_parent_nids)) {
+              $latest_revision->set('field_include_parent_org_search', 1);
+            }
+            $latest_revision->setSyncing(TRUE);
```

Comment:
IMO we should set a revision log message that says 'Deploy hook setting field_org_no_search_filter and  field_include_parent_org_search to 1'

Reply:
Added